### ISSUE TITLE
neko is releasing whats seems like beta versions atualized more frequently in this tg group https://t.me/nekox_broken

### ISSUE THREAD

**Issue by y1111** [NekoX-Dev/NekoX#1041]
neko is releasing whats seems like beta versions atualized more frequently in this tg group https://t.me/nekox_broken

they call NekoX - ghetto editon . - Different package name, able to coexist with official NekoX (from pinned msg there)

by today last version posted is 10.6.1 ,posted january 23

_Originally posted by @y1111 in https://github.com/NekoX-Dev/NekoX/issues/1028#issuecomment-1935884462_
            

opening issue , because it seems , people here are not aware of where the current build are being posted

**Comment by y1111** [comment]
link https://t.me/nekox_broken

**Comment by dic1911** [comment]
1.
> neko is releasing...

No, not me or any NekoX dev ever reference self as neko

2. It doesn't look good to open an issue just for showing the existence of another fork.

3. In case you haven't realize, I'm the one who work on ghetto edition and I rather you haven't post anything about it here.

**Comment by y1111** [comment]
u mean that group gheto edition is not the same dev group of this one here ?

**Comment by y1111** [comment]
so maybe i didnt remenber or didnt noticed that until now . so if its diferent people dev , other fork so i should rephrase it to like this other option here is a maintained forkof nekoX . is it more corect / or how should be ?



> > neko is releasing...
> 
> No, not me or any NekoX dev ever reference self as neko
> 
> 2. It doesn't look good to open an issue just for showing the existence of another fork.
> 3. In case you haven't realize, I'm the one who work on ghetto edition and I rather you haven't post anything about it here.

**Comment by y1111** [comment]
i just saw in your page u have a fork and is posting releases here too right https://github.com/dic1911/NekoX

**Comment by ArbitraryFox** [comment]
> i just saw in your page u have a fork and is posting releases here too right https://github.com/dic1911/NekoX

Boi, the dev just said he would not want you to mention the app/links here, and not to use the issue tickets to tell everyone about the existence of another fork. Stop. Just stop.

**Comment by y1111** [comment]
> ArbitraryFox

oh .... ok .  im not seeing how to delete an issue  , how to ? . i will remove the links on the posts so . should i ? or remove all the posts ?
but why not meantion a maintained fork . people here and in fdroid store users all think is abandonware by now.

**Comment by y1111** [comment]
or could i just leave like . "guys search for nekoX ghetto on tg" , and not put any link  .  or u prefer just delete remove all mention @dic1911  ?

**Comment by dic1911** [comment]
@y1111 

> oh .... ok . im not seeing how to delete an issue , how to ? ...

By editing your posts, that's the best you can do now.

> remove all mention

this

**Comment by y1111** [comment]
> @y1111
> 
> > oh .... ok . im not seeing how to delete an issue , how to ? ...
> 
> By editing your posts, that's the best you can do now.
> 
> > remove all mention
> 
> this

is that now good enough . just telling people to search for the tg ? can the name ghetto be mentioned ? (i delete this msg later
)
